visidata/menu.py

```diff
@@ -786,6 +790,7 @@ def runMenu(vd):
     vd.draw_all()
     sheet.execCommand(currentItem.longname)
 
+main_menu = {'f': 'File', 'e': 'Edit', 'v': 'View', 'c': 'Column', 'r': 'Row', 'd': 'Data', 'p': 'Plot', 's': 'System', 'h': 'Help'}
```

Comment:
Hm, this can work for now, but maybe we should have a more programmatic way of generating this list, esp since plugins can add toplevel menu items.